tests/unit/s2n_self_talk_session_id_test.c

```diff
@@ -272,9 +299,17 @@ int main(int argc, char **argv)
     char buffer[256];
     int bytes_read;
     int shutdown_rc = -1;
+    uint64_t now;
     uint8_t session_id_from_server[MAX_KEY_LEN];
     uint8_t session_id_from_client[MAX_KEY_LEN];
 
+     /* aes keys. Used for session ticket/session data encryption. Taken from test vectors in https://tools.ietf.org/html/rfc5869 */
+    uint8_t ticket_key_name[16] = "2018.07.26.15\0";
```

Comment:
you are only using this as a `char *` below, so retype it?

also:

```
char ticket_key_name[] = "2018.07.26.10";
```

Reply:
Also update in s2nd.c

Reply:
Changes are reverted as it conflicts with function definition here https://github.com/awslabs/s2n/blob/master/tls/s2n_config.c#L738-L741

As `s2n_config_add_ticket_crypto_key` is called outside https://github.com/awslabs/s2n/blob/master/api/s2n.h#L160. I turned out to revert the change. 

ticket_key_name is similarly defined here https://github.com/awslabs/s2n/blob/master/bin/s2nd.c#L147